responder/interface.go

```diff
@@ -0,0 +1,13 @@
+package responder
+
+type dataLogger interface {
+	LogData() map[string]interface{}
+}
+
+type coder interface {
+	Code() int
+}
+
+type messager interface {
```

Comment:
```suggestion
type messenger interface {
```
just a suggestion

Reply:
Ah no this is by design as the `Messager` interface directly suggests the main method it exposes (`Message()`)

Reply:
cool, fair enough